examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -1,57 +1,89 @@
 """
-Extract signals and plot a connectome for seed-based atlases
-============================================================
+Extract signals and plot a connectome for atlas-based spheres
+=============================================================
 
-This example shows how to extract signals from spherical seed-based
-atlases such as Power-264 atlas (Power, 2011) and Dosenbach-160 (Dosenbach,
-2010). We estimate connectome using sparse inverse covariance.
+This example shows how to extract preprocessed signals from spherical regions
+centered on coordinates from Power-264 atlas [1] and Dosenbach-160 [2].
+We estimate connectome using sparse inverse covariance, to recover
+the functional brain networks structure.
 
-Dosenbach N.U., Nardos B., et al. "Prediction of individual brain maturity
+References
+----------
+[1] Power, Jonathan D., et al. "Functional network organization of the
+human brain." Neuron 72.4 (2011): 665-678.
+
+[2] Dosenbach N.U., Nardos B., et al. "Prediction of individual brain maturity
 using fMRI.", 2010, Science 329, 1358-1361.
 
-Power, Jonathan D., et al. "Functional network organization of the
-human brain." Neuron 72.4 (2011): 665-678.
 """
 
+###############################################################################
+# Atlases loading
+# ---------------
+
+###############################################################################
+# We fetch the coordinates of Power atlas
+from nilearn import datasets
+
+power = datasets.fetch_coords_power_2011()
+print(power.keys())
+
+###############################################################################
+# and the coordinates, regions labels and networks names of the Dosenbach
+# atlas.
+dosenbach = datasets.fetch_coords_dosenbach_2010(ordered_regions=True)
+print(dosenbach.keys())
```

Comment:
I'd rather "transpose" this code: do all the code for one atlas, with a lot of comments, and then add a section at the end that does the same thing for the dosenbach atlas.